diff --git a/build.gradle b/build.gradle
index 1cb3075d1..404f6aee2 100644
--- a/build.gradle
+++ b/build.gradle
@@ -12,6 +12,13 @@ buildscript {
 
 plugins {
     id("net.ltgt.errorprone") version "5.1.0" apply false
+    id 'org.cyclonedx.bom' version '2.3.1'
+}
+
+cyclonedxBom {
+    projectType = "application"
+    outputName = "linstor-server"
+    outputFormat = "json"
 }
 
 apply plugin: 'java'
