diff --git a/build.gradle b/build.gradle
index dad724cd9..858f4dc3b 100644
--- a/build.gradle
+++ b/build.gradle
@@ -15,6 +15,16 @@ buildscript {
     }
 }
 
+plugins {
+    id 'org.cyclonedx.bom' version '2.3.1'
+}
+
+cyclonedxBom {
+    projectType = "application"
+    outputName = "linstor-server"
+    outputFormat = "json"
+}
+
 apply plugin: 'java'
 apply plugin: 'eclipse'
 apply plugin: 'idea'
